### DISCUSSION TITLE
Example pipeline with a Jenkins Shared Library in external repo

### DISCUSSION THREAD

**Discussion by jimklimov** [lppedd/idea-jenkins-pipeline#2] [General]
As asked in the IDEA issue discussion, an example (note: with a scripted pipeline!) was posted and detailed there and referenced here:

https://youtrack.jetbrains.com/issue/IDEA-320924/IDEA-does-not-handle-Groovy-call-class-methods-call-operator-for-context-help-and-auto-completion#focus=Comments-27-7360970.0-0

**Comment by lppedd** [comment]
Thanks @jimklimov. I'll try to repro and check that code completion works as best as possible with the plugin installed.
Unfortunately being it based on the Groovy support it has more or less the same limitations, but I can "inject" fake method references.
